.github/workflows/github-actions-es.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+name: GitHub Actions ES
+run-name: ${{ github.actor }} is push code to main 🚀
+on:
+  push:
+    branches: [ "main", "ci" ]
```

Comment:
does it mean that it only works on the ci branch merge into main branch?

Reply:
It will only be triggered when code is submitted to the main or ci branch.